docs/Performance.md

```diff
@@ -39,3 +53,19 @@ For an average CPU usage of 5%
 
 This shows that the collector performs better when it is made to ingest bigger
 log entries (which is expected due to less overhead coming from timestamp parsing etc.).
+
+#### Memory usage guidelines
+
+#### Benchmark - memory usage for particular average message size and EPS
+
+Measured memory usage (in MB) for particular Events Per Second (EPS) average message size.
+
+|          |  100B  |  512B  |  1KB   |  5KB   |  10KB  |
+|:--------:|:------:|:------:|:------:|:------:|:------:|
+| **EPS**  |   -    |   -    |   -    |   -    |   -    |
+| **100**  | 113.14 | 116.16 | 117.1  | 116.99 | 112.59 |
+| **200**  | 115.16 | 118.55 | 116.8  | 119.67 | 127.02 |
+| **500**  | 118.24 | 121.79 | 122.78 | 127.87 | 142.73 |
+| **1000** | 121.6  | 126.75 | 127.94 | 140.11 | 106.82 |
+| **1500** | 128.54 | 131.9  | 137.69 | 95.21  | 113.89 |
+| **2000** | 130.62 | 125.27 | 144.59 | 98.62  | 134.61 |
```

Comment:
looks unstable and little bit confusing. How did we measured that?

Reply:
I'm merging this in. When we get this clarified I'll update this table.